keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -1061,28 +1062,90 @@ Details to consider:
 
 #### CEL Expression Composition
 
+##### Use Cases
+
+###### Code re-use for complicated expressions
+
+A CEL expression may not be computationally expensive, but could still be
+intricate enough that copy-pasting could prove to be a bad decision later on
```

Comment:
Worth mentioning especially after formatted message in place, we would expect more copy-pasting?